lib/gateways/plan-gateway.js

```diff
@@ -63,7 +63,7 @@ export default class PlanGateway {
 
     const request = {
       TableName: this.tableName,
-      IndexName: 'name_idx',
+      IndexName: 'NamesIndex',
```

Comment:
We could make this an environment variable maybe? ...but pretty unlikely to ever change once set up.

Reply:
yeah I think that's cool to leave as-is tbh